macros/upload_exposures.sql

```diff
@@ -58,13 +62,15 @@
                     '{{ run_started_at }}', {# run_started_at #}
                     '{{ exposure.name | replace("'","\\'") }}', {# name #}
                     '{{ exposure.type }}', {# type #}
-                    parse_json('{{ tojson(exposure.owner) | replace("'","\\'") }}'), {# owner #}
+                    '{{ tojson(exposure.owner) | replace("'","\\'") }}', {# owner #}
                     '{{ exposure.maturity }}', {# maturity #}
                     '{{ exposure.original_file_path | replace('\\', '\\\\') }}', {# path #}
-                    """{{ exposure.description | replace("'","\\'") }}""", {# description #}
+                    '{{ exposure.description | replace("'","\\'") }}', {# description #}
                     '{{ exposure.url }}', {# url #}
                     '{{ exposure.package_name }}', {# package_name #}
-                    {{ tojson(exposure.depends_on.nodes) }} {# depends_on_nodes #}
+                    '{{ tojson(exposure.depends_on.nodes) }}', {# depends_on_nodes #}
+                    '{{ tojson(exposure.meta) | replace("'","\\'") }}' {# meta #}
```

Comment:
```suggestion
                    parse_json('{{ tojson(exposure.meta) | replace("'","\\'") }}') {# meta #}
```

As this is a key: value object, I think it needs `parse_json`